app/admin/quests/create/page.tsx

```diff
@@ -0,0 +1,1049 @@
+        });
+      }
+    });
+    setCurrentPage((prev) => prev + 1);
+  }, [steps]);
+
+  return (
+    <div className={styles.layout_screen}>
+      <ProgressBar currentStep={currentPage} totalSteps={4} />
+      <p className={styles.screenHeadingText}>Create a new quest</p>
+      {currentPage === 1 ? (
+        <>
+          <InputCard>
+            <div className="flex flex-col gap-8 w-full">
+              <p className={styles.cardHeading}>{headingText}</p>
+              <Textinput
+                onChange={(e) => {
+                  setQuestInput((prev) => ({
+                    ...prev,
+                    name: e.target.value,
+                  }));
+                  setBoostInput((prev) => ({
+                    ...prev,
+                    name: e.target.value,
+                  }));
+                }}
+                value={questInput.name}
+                name="name"
+                label="Quest Name"
+                placeholder="Quest Name"
+              />
+              <Textinput
+                onChange={handleQuestInputChange}
+                value={questInput.desc}
+                name="desc"
+                label="Description"
+                placeholder="Quest Description"
+                multiline={4}
+              />
+              <div className="w-full flex justify-between gap-4">
+                <div className="flex-1 w-full">
+                  <Dateinput
+                    onChange={(e) => {
+                      setStartTime(e.target.value);
+                    }}
+                    value={startTime}
+                    label="Start Date"
+                    placeholder="Start Date"
+                    name="start_time"
+                  />
+                </div>
+                <div className="flex-1 w-full">
+                  <Dateinput
+                    onChange={(e) => {
+                      setEndTime(e.target.value);
+                    }}
+                    value={endTime}
+                    label="End Date"
+                    placeholder="End Date"
+                    name="expiry"
+                  />
+                </div>
+              </div>
+              <div className="flex flex-row w-full gap-2 bg-gray-300 p-2 rounded-xl">
+                {CATEGORY_OPTIONS.map((category) => (
+                  <div
+                    onClick={() => {
+                      setQuestInput((prev) => ({ ...prev, category }));
+                    }}
+                    key={category}
+                    className="py-3 px-5 rounded-xl w-fit"
+                    style={{
+                      cursor: "pointer",
+                      backgroundColor:
+                        questInput.category === category
+                          ? "#ffffff"
+                          : "#29282B",
+                      color:
+                        questInput.category === category
+                          ? "#29282B"
+                          : "#ffffff",
```

Comment:
I don't like inline css overall, on top of that here don't we have `theme.ts` to not write code colors in string ?